indexing-service/src/main/java/io/druid/indexing/common/task/CompactionTask.java

```diff
@@ -348,10 +402,11 @@ private static DataSchema createDataSchema(
       final Boolean isRollup = pair.lhs.getMetadata().isRollup();
       return isRollup != null && isRollup;
     });
+
     final GranularitySpec granularitySpec = new ArbitraryGranularitySpec(
-        new NoneGranularity(),
+        GranularityType.NONE.getDefaultGranularity(),
```

Comment:
`Granularities.NONE` is a simpler way of writing this.

Reply:
Fixed.